### PR TITLE
a script to benchmark our obfuscators

### PR DESCRIPTION
**Pull request by trevj**
Depends on:
https://github.com/uProxy/uproxy-lib/pull/291

Yields graphs like this:
obfuscator benchmarks

### CODE REVIEW

File: testing/run-scripts/benchmark_transformers.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        flood_ip, '1224']) != 0:
+      raise Exception('nc failed, proxy probably did not start')
+    end = time.time()
+
+    print('** benchmarking...')
+
+    elapsed = round(end - start, 2)
+    result = int((FLOOD_SIZE_MB / elapsed) * 1000)
+
+    print('** throughput for ' + transformer + ': ' + str(result) + 'K/sec')
```
Comment (mollyling) [inline testing/run-scripts/benchmark_transformers.py]: KB/sec?

File: testing/run-scripts/benchmark_transformers.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+import argparse
+import csv
+import io
+import subprocess
+import sys
+import time
+import urllib.parse
+
+FLOOD_SIZE_MB = 64
+FLOOD_MAX_SPEED = '5M'
```
Comment (mollyling) [inline testing/run-scripts/benchmark_transformers.py]: Perhaps standardize this a bit?  '64M' to include the unit suffix, or 5 to exclude it (putting the +'M' in the call to subprocess.check_output().

**Comment by mollyling** [comment]
:+1:

File: testing/run-scripts/benchmark_transformers.py (hunk 1 above)
Comment (trevj) [inline testing/run-scripts/benchmark_transformers.py]: Done.

File: testing/run-scripts/benchmark_transformers.py (hunk 2 above)
Comment (trevj) [inline testing/run-scripts/benchmark_transformers.py]: Great idea, thanks.

**Comment by trevj** [comment]
Thanks @blanu! I removed sleep and now we're good.

**Pull request merged**